models/balancer/ethereum/balancer_v2_ethereum_pools_tokens_weights.sql

```diff
@@ -0,0 +1,62 @@
+{{
+    config(
+        schema="balancer_v2_ethereum",
+        alias='pools_tokens_weights',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['pool_id', 'token_address'],
+        post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "project",
+                                    "balancer",
+                                    \'["metacrypto", "jacektrocinski"]\') }}'
+    )
+}}
+
+--
+-- Balancer v2 Pools Tokens Weights
+--
+SELECT
+    registered.`poolId` AS pool_id,
```

Comment:
can you remove the '`' usage on all models? it's not a must on spark & will help prepare for future-state architecture changes

Reply:
Sure thing, removed the tick marks from the code.